CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 ## master
 
-## [v5.6.0][v5.6.0] (July 22, 2022)
+## [v5.6.1][v5.6.1] (December 14, 2022)
+
+* Dependency updates
```

Comment:
Either don't mention this or mention what dependencies were updated.

Reply:
Removing dependencies as we don't need to mention that.